extension/src/openvic-extension/classes/GUILabel.cpp

```diff
@@ -31,12 +31,18 @@ String const& GUILabel::get_substitution_marker() {
 	return SUBSTITUTION_MARKER;
 }
 
+String const& GUILabel::get_flag_marker() {
```

Comment:
(Probably me lacking context)
Why is this a method instead of a constant value?

Reply:
`static` variables outside of functions are initialised at the very beginning of the program (before the `main` function is called), but `static` variables inside functions are only initialised when the function is first called/the line defining the variable is first reached.
We can't initialise any `String`s or `StringName`s before Godot/GDExtension has run its own setup code, otherwise it causes crashes/breaks the library loading process (our code is compiled as a library that is dynamically loaded by the Godot Editor or the exported Godot program when they start up). So any constants like `FLAG_MARKER` have to be wrapped in a function to delay their initialisation.